sympy/core/numbers.py

```diff
@@ -2038,6 +2053,154 @@ def as_coeff_Add(self, rational=False):
         return self, S.Zero
 
 
+class DecimalRational(Rational):
+    """Represents rationals that print as decimals
+
+    Examples
+    ========
+
+    >>> from sympy import DecimalRational, latex
+    >>> DecimalRational(1,10)
+    0.100000000000000
+    >>> latex(DecimalRational('.1'))
+    '0.1'
+
+    """
+    is_DecimalRational = True
+
+    @cacheit
+    def __new__(cls, *args):
```

Comment:
Is it necessary to duplicate all this code on `DecimalRational`? It would be better if it weren't duplicated. The way it is here, any updates to Rational will need to also be made to DecimalRational. We would also need to make sure all this code is tested.

Reply:
I added tests to illustrate why all the changes I made were necessary.  Maybe there is a more elegant way to do arithmetic with DecimalRationals without them getting moved back to Rational so that the tests still pass.

Reply:
Giving DecimalRationals a higher operation priority is the more elegant way to do this so that all this code from Rational doesn't need to be duplicated